src/i18n/locales/en_US/epg.json

```diff
@@ -0,0 +1,4 @@
+{
+  "on_channel": "On {{name}}",
+  "start_from_beginning": "Watch from begin"
```

Comment:
This doesn't sound quite right (not sure why it's not reflected on the preview url)

Reply:
Yes, you're right! According to @marcovandeveen it should be `Watch from start`